pytest_tldr.py

```diff
@@ -1,17 +1,24 @@
 # -*- coding: utf-8 -*-
 from __future__ import print_function
-
 import platform
 import sys
 import time
 
 import pluggy
 import py
 import pytest
-from _pytest.main import (
-    EXIT_OK,
-    EXIT_TESTSFAILED,
-)
+try:
+    from _pytest.main import ExitCode
```

Comment:
(Just saw this being announced on twitter, congrats!)

This should use the public API:

```python
from pytest import ExitCode
```

Reply:
Thanks for the suggestion! I've just added that cleanup as part of #19.